src/odemis/model/_metadata.py

```diff
@@ -271,3 +271,5 @@
 # Fastem: Parameters used for stitching and reconstruction of 3D volumes
 MD_SLICE_IDX = "Index of slice in volume stack"  # int
 MD_FIELD_SIZE = "Average field of view of a megafield"  # tuple (px, px)
+
+MD_CHROMATIC_COR = "Chromatic cor"
```

Comment:
Please give a better string, it should be "user-friendly". Also add a comment of the format of the data it contains.